src/pyscipopt/scip.pxd

```diff
@@ -1660,6 +1686,21 @@ cdef extern from "scip/pub_lp.h":
     int SCIPcolGetIndex(SCIP_COL* col)
     SCIP_Real SCIPcolGetObj(SCIP_COL *col)
 
+cdef extern from "scip/lp.h":
+    SCIP_Real SCIPlpGetObjNorm(SCIP_LP* lp)
+    # void SCIPlpRecalculateObjSqrNorm(SCIP_SET* set, SCIP_LP* lp)
+    SCIP_Real SCIProwGetNorm(SCIP_ROW* row)
+    SCIP_Real SCIProwGetDualsol(SCIP_ROW* row)
+    int SCIProwGetAge(SCIP_ROW* row)
+    SCIP_Real SCIProwGetLPActivity(SCIP_ROW* row,
+                                   SCIP_SET* set,
+                                   SCIP_STAT* stat,
+                                   SCIP_LP* lp)
+    SCIP_Real SCIProwGetObjParallelism(SCIP_ROW* row,
+                                       SCIP_SET* set,
+                                       SCIP_LP* lp)
+    SCIP_Real SCIPcolGetObj(SCIP_COL *col)
+
```

Comment:
Could you explain where this is used for your intended functionality?

Reply:
It has none. I cleaned it up. :)